_includes/posts.html

```diff
@@ -0,0 +1,9 @@
+{% for post in site.posts %} 
+{% unless post.categories contains "notice" %} 
+{% if post.pinned == true %}
+{% unless page.url == post.url %}
```

Comment:
Trudno za tym podążyć, już chyba lepiej żeby to było jako jeden warunek.

Reply:
chyba tak nie działa w jekyllu

Reply:
W zasadzie wygląda na to, że wśród operatorów brakuje negacji https://shopify.github.io/liquid/basics/operators/